includes/media_bynder.admin.inc

```diff
@@ -550,20 +550,13 @@ function media_bynder_search_results($search, $selected_facets, $meta_properties
 
     if ($result) {
         $arr_json = json_decode($result, TRUE);
-
-        //Set total count;
-        if(isset($arr_json['count']['brandId'])){
-            $total_count = 0;
-            foreach($arr_json['count']['brandId'] as $brand_count){
-                $total_count += $brand_count;
-            }
-            $arr_json['totalNrOfResults'] = $total_count;
+        if (isset($arr_json['count']['total'])) {
+            $arr_json['totalNrOfResults'] = $arr_json['count']['total'];
```

Comment:
The fallback uses $arr_json['count']['brandId']: so it wouldn't help as a fallback for the missing array  $arr_json['count']. Either $arr_json['count'] exist with all it's subarrays or it doesn't. Never just $arr_json['count']['brandId'] without $arr_json['count']['total']. That's why it will never happen that $arr_json['count']['brandId'] exists but $arr_json['count']['total'] doesn't.